pkg/importer/http-datasource.go

```diff
@@ -112,6 +115,10 @@ func (hs *HTTPDataSource) Info() (ProcessingPhase, error) {
 		klog.Errorf("Error creating readers: %v", err)
 		return ProcessingPhaseError, err
 	}
+	// special case to support glance
```

Comment:
This isn't actually a workaround to support glance: it's applicable to any HTTP source which requires a custom header, because we can't pass that header to qemu-img, and therefore we need to download directly with curl.

This is the same whether glance supports or doesn't support byte ranges (which it doesn't).

Reply:
You are right will remove this comment. TNX